[CI]feat:Add NPU CI action and fallback SFT's e2e test defaults to FSDP1

Crispig opened the pull request:
### Checklist Before Starting

- [done] Search for similar PR(s).

### What does this PR do?

Mirror the CI for VeRL to run on the NPU and fallback the e2e test of the SFT to FSDP1, as the NPU is not currently adapted for FSDP2

### Specific Changes

Add `.github/workflows/e2e_ascend.yml`
Change `tests/e2e/sft/run_sft.sh`

### Checklist Before Submitting

- [ done ] Read the [Contribute Guide](https://github.com/volcengine/verl?tab=readme-ov-file#contribution-guide).
- [ done ] Apply [pre-commit checks](https://github.com/volcengine/verl?tab=readme-ov-file#code-linting-and-formatting).

vermouth1992 commented:
Could you move https://github.com/volcengine/verl/blob/main/.github/workflows/disabled/e2e_ascend.yml instead of creating a new one? Thanks.

Crispig commented:
> Could you move https://github.com/volcengine/verl/blob/main/.github/workflows/disabled/e2e_ascend.yml instead of creating a new one? Thanks.

OK, I moved the e2e_ascend.yml in disable via git rm and git mv and modified it under workflow.
